node/pkg/logscribeconsumer/app.go

```diff
@@ -1,33 +1,46 @@
-package zeropglog
+package logscribeconsumer
 
 import (
 	"context"
 	"encoding/json"
+	"net/http"
 	"os"
 	"strings"
 	"time"
 
-	"bisonai.com/orakl/node/pkg/db"
 	errorsentinel "bisonai.com/orakl/node/pkg/error"
+	"bisonai.com/orakl/node/pkg/utils/request"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 )
 
-func New(options ...AppOption) *App {
+func New(options ...AppOption) (*App, error) {
 	consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
 
 	c := &AppConfig{
-		StoreInterval: DefaultLogStoreInterval,
-		Buffer:        DefaultBufferSize,
+		StoreInterval:    DefaultLogStoreInterval,
+		Buffer:           DefaultBufferSize,
+		LogscribeEnpoint: DefaultLogscribeEnpoint,
 	}
 	for _, option := range options {
 		option(c)
 	}
-	return &App{
-		StoreInterval: c.StoreInterval,
-		buffer:        make(chan map[string]any, c.Buffer),
-		consoleWriter: consoleWriter,
+
+	if c.Service == "" {
+		return nil, errorsentinel.ErrLogscribeConsumerServiceNotProvided
 	}
+	if !isLogLevelValid(map[string]any{"level": c.Level}) {
```

Comment:
`isLogLevelValid` func is intended for filtering out lowLevel log
please double check the function code and update this initialization

Reply:
yea i misunderstood the point of passing the log level to the app instance. Should be fixed now!